src/style.css

```diff
@@ -0,0 +1,143 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px 19px 36px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  margin: 0 16px 16px;
+}
+
+.item-card__name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.item-card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars {
+  margin: 0 16px 24px;
+}
+
+.element-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.element-card__stars {
+  display: flex;
+}
+
+.element-card__star {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.element-card__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.element-card__stars--not-active :nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
Reuse `stars` block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change `stars--4` modifier to `stars--2` or other - number of selected stars should change correctly.